qemu/qmp.go

```diff
@@ -883,11 +891,31 @@ func (q *QMP) ExecHotplugMemory(ctx context.Context, qomtype, id, mempath string
 }
 
 // ExecutePCIVSockAdd adds a vhost-vsock-pci bus
-func (q *QMP) ExecutePCIVSockAdd(ctx context.Context, id, guestCID string) error {
+func (q *QMP) ExecutePCIVSockAdd(ctx context.Context, id, guestCID, vhostfd string, disableModern bool) error {
 	args := map[string]interface{}{
 		"driver":    VHostVSockPCI,
 		"id":        id,
 		"guest-cid": guestCID,
 	}
+
+	if vhostfd != "" {
```

Comment:
Could you please change `vhostfd` to `*os.File` instead? That would align with other part of govmm that fd is always passed as `*os.File`

Reply:
nop, because `vhostfd` can be a name, the same `fdname` passed to `ExecuteGetFD`

Reply:
oops, I missed that. Sorry for the noise.

Reply:
@devimc don't we want to enforce the `vhostfd` parameters to be passed all the time? I'm asking because using this function without passing the vhost file descriptor is potentially racy as there is no guarantee the `CID` will not be already used by some other process.

Reply:
I agree, let's change it :smile: